src/transaction.ts

```diff
@@ -284,8 +288,31 @@ class Transaction extends DatastoreRequest {
    *   if (err) {
    *     // Error handling omitted.
    *   }
+   *   const ancestorKey = datastore.key(['ParentCompany', 'Alphabet']);
    *
-   *   const query = transaction.createQuery('Company');
+   *   const query = transaction.createQuery('Company').hasAncestor(ancestorKey);
+   *
+   *   query.run((err, entities) => {
+   *     if (err) {
+   *       // Error handling omitted.
+   *     }
+   *
+   *     transaction.commit((err) => {
+   *       if (!err) {
+   *         // Transaction committed successfully.
+   *       }
+   *     });
+   *   });
+   * });
+   *
+   * //Run the query inside the transaction.with namespace
```

Comment:
Probably want a space inserted between the comment characters and the comment itself.